core/src/test/java/com/linecorp/armeria/common/util/ExceptionsTest.java

```diff
@@ -66,4 +77,31 @@ public void testTraceText() {
         assertThat(trace).startsWith(Exception.class.getName() + ": foo\n" +
                                      "\tat " + ExceptionsTest.class.getName());
     }
+
+    @Test
+    public void testIsExpected() {
+        final boolean expected = !Flags.verboseExceptions();
```

Comment:
Just wondering if we can just make this as `true`.(not using `!Flags.verboseExceptions()`)

Reply:
There was actually a bug in `Exceptions.isExpected()` where a wrong boolean value is returned when `Flags.verboseExceptions()` is `true`.